resources/styleheets/base/_login.scss

```
@@ -0,0 +1,214 @@
+body.login{
```

```suggestion
body.login {
```
space between the brackets